internal/target/repository_credential_source_test.go

```diff
@@ -44,9 +44,12 @@ func (h hooks) VetCredentialSources(ctx context.Context, cls []*target.Credentia
 	return targettest.VetCredentialSources(ctx, cls, creds)
 }
 
+func init() {
+	target.Register(targettest.Subtype, hooks{}, globals.TcpTargetPrefix)
+}
+
 func TestRepository_SetTargetCredentialSources(t *testing.T) {
 	ctx := context.Background()
-	target.Register(targettest.Subtype, hooks{}, globals.TcpTargetPrefix)
 
```

Comment:
I had to move this to make these hooks available to all external tests. Without this, ListTarget fails to read the test target type from the DB.

Reply:
It might be good to move this `init` into a `ext_test.go` file. And we should probably rename any test files in this package that use `package target_test` to have `_ext_test.go` as a suffix. I think @jimlambrt started that pattern of using `_ext_test` and it would probably help make it easier to identify which files contain tests that will be in the external package.

Reply:
Good idea, I'll just move this test into the `_ext_test.go` file.